src/main/java/com/example/flipcardsapp/FlipCard/FlipCardImp.java

```diff
@@ -0,0 +1,54 @@
+package com.example.flipcardsapp.FlipCard;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+public class FlipCardImp {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    @OneToOne(cascade = CascadeType.ALL)
+
+    private FlipCardFront flipCardFront;
+    @OneToOne(cascade = CascadeType.ALL)
+    private FlipCardBack flipCardBack;
+
+    public FlipCardImp() {
+    }
+
+    public FlipCardImp(FlipCardFront flipCardFront, FlipCardBack flipCardBack) {
+        this.flipCardFront = flipCardFront;
+        this.flipCardBack = flipCardBack;
+    }
+
+    public FlipCardFront getFlipCardFront() {
+        return flipCardFront;
+    }
+
+    public FlipCardBack getFlipCardBack() {
+        return flipCardBack;
+    }
+
+    @Override
```

Comment:
check how to implement this methods in hibernate
https://vladmihalcea.com/the-best-way-to-implement-equals-hashcode-and-tostring-with-jpa-and-hibernate/